### PR TITLE
Add demo using a custom `drawer-toggle-attribute`

### LINKED ISSUE #71: drawerToggleAttribute
if you customize this property, it won't hide the toggle element in narrow layout.

### PR DESCRIPTION
**Pull request by blasten**
So we can close #71

### CODE REVIEW

File: demo/custom-drawer-toggle-attribute.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      }
+
+      paper-button {
+        color: white;
+        margin: 10px;
+        background-color: var(--google-blue-700);
+        white-space: nowrap;
+      }
+
+      paper-drawer-panel:not([narrow]) [custom-toggle] {
```
Comment (frankiefu) [inline demo/custom-drawer-toggle-attribute.html:45]: Add a comment here so user knows this is needed for showing/hiding the toggle element when using custom `drawer-toggle-attribute`.

File: demo/custom-drawer-toggle-attribute.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    <script>
+      (function(global) {
+
+        'use strict';
+
+        var DRAWER_ATTR = 'right-drawer';
+
+        var pdp = document.getElementById('paperDrawerPanel');
+
+        global.flipDrawer = function() {
```
Comment (frankiefu) [inline demo/custom-drawer-toggle-attribute.html:74]: Do we need this for this demo?  I don't see the button that triggers `flipDrawer`

**Comment by goldieep** [comment]
@frankiefu can this be merged I demand this to be merged soon

**Comment by blasten** [comment]
closing this

**Pull request closed without merging**